### DISCUSSION TITLE
Placeholder not working

### DISCUSSION THREAD

**Discussion by QueenAngus** [Zrips/CMI#3702] [Enhancement]
**Description of issue:**
Placeholders still do not work in schedules
<img width="488" alt="Screen Shot 2020-07-12 at 7 30 45 PM" src="https://user-images.githubusercontent.com/11791722/87241333-3f92ed00-c476-11ea-8adc-0fd2f29dc821.png">

---

---

**Cmi Version (using`/cmi version`):** 
8.7.1.1
**Server Type (Spigot/Paperspigot/etc):** 
Paper
**Server Version (using `/ver`):** 
1.13.2

**Comment by CloudeLecaw** [comment]
May the voteparty placeholder broke itself?

**Comment by QueenAngus** [comment]
No it is all placeholders

**Comment by CloudeLecaw** [comment]
Hm we are using broadcast schedulers with placeholders of other plugins like a lottery plugin too on our server, they all work. We use latest cmi version + lastest paper 1.16.1
So it looks like an issue with paper 1.13.2 or with incompatibility of the plugin itself if you say no placeholders working right...

**Comment by Zrips** [comment]
For time being placeholders in schedule will not work. I need to rethink its behavior in general as we have 2 situations with placeholders where on one hand we need to translate them before performing commands and then on another hand we need to avoid translating them and pass over placeholders to the command. Both of them cant really exist at same time but at same time we need both behaviors to get most out of it.
